src/content/docs/features/authentication.mdx

```diff
@@ -50,31 +93,45 @@ For teams or contractors, Social Sign-On offers a fast and convenient way to acc
 
 ## Multi-Factor Authentication (MFA)
 
-MFA adds an additional layer of security by requiring two or more verification steps during login. Typically, it combines something you know (like a password) and something you have (like a mobile app code).
+- The <a href="https://en.wikipedia.org/wiki/Multi-factor_authentication" target="_blank" rel="noopener noreferrer"> MFA </a> adds an additional layer of security by requiring two or more verification steps during login. Typically, it combines something you know (like a password) and something you have (like a mobile app code).
 
 ### Enabling MFA
 
 1. **Go to Account Settings**
+
    - Locate the **Multi-Factor Authentication** option.
 
-   ![MFA Settings](/platform/authentication/mfa.png)
+   <img
+     src="/platform/authentication/mfa.png"
+     width="800"
+     style={{ borderRadius: "8px" }}
+   />
 
 2. **Enable MFA**
+
    - Click **Enable MFA** to generate a QR code.
 
-   ![Enable MFA](/platform/authentication/mfa-enable.png)
+   <img
+     src="/platform/authentication/mfa-enable.png"
+     width="800"
+     style={{ borderRadius: "8px" }}
+   />
 
 3. **Scan and Verify**
-   - Use an authentication app (e.g., Google Authenticator) to scan the QR code. 
+   - Use an authentication app (e.g., Google Authenticator) to scan the QR code.
    - Verify your MFA setup by entering the code from the app.
 
 ## Social Sign-On
 
-Social Sign-On lets users log in with existing accounts such as Google or GitHub.
-This feature is particularly useful for teams that prefer using familiar credentials instead of creating new ones.
+- The <a href="https://en.wikipedia.org/wiki/Social_login" target="_blank" rel="noopener noreferrer"> Social Sign-On </a> lets users log in with existing accounts such as Google or GitHub.
+- This feature is particularly useful for teams that prefer using familiar credentials instead of creating new ones.
 
-![Social Sign On](/platform/authentication/social-sign-on.png)
+<img
```

Comment:
The image looks too zoomed. I would reduce the zooming to keep it in sync with other images.